onelogin/get.go

```diff
@@ -216,3 +199,52 @@ func Get(app, provider string) (*awsprovider.Credentials, error) {
 
 	return &creds, nil
 }
+
+// getDevice returns the MFA device used by the user. If there is
+// more than one available the user is prompted which one should
+// be used.
+func getDevice(devices []Device) (deviceID, deviceType string, err error) {
+	if len(devices) == 0 {
+		// this should never happen
```

Comment:
I guess `len(devices)` can be zero if MFA is disabled, unless I am missing something. In any case, as long as we have this comment here, can we please make it consistent with the rest?

```go
// This should never happen.
```

Thanks.